worker/workercmd/worker_linux.go

```diff
@@ -132,6 +133,10 @@ func (cmd *WorkerCommand) gdnRunner(logger lager.Logger) (ifrit.Runner, error) {
 		"--no-image-plugin",
 	}
 
+	for _, dnsServer := range cmd.Garden.DNSServers {
+		gdnServerFlags = append(gdnServerFlags, "--dns-server", dnsServer)
```

Comment:
Was it not possible to set the list of DNS servers in Guardian before this?

Reply:
goood question: it would be automagically forwarded through https://github.com/concourse/concourse/blob/aee8c5cbde7982361d109bc28a9db8a68e6f0664/worker/workercmd/worker_linux.go#L274-L277

we had to have that explicitly because the library that takes the env vars and fills the `cmd.*` structs unsets from the env, which means that just because we added there so support `containerd`, that'd helper would not get the var anymore